rollup.config.js

```diff
@@ -10,7 +10,7 @@ const plugins = [
   json(),
   commonjs(),
   esbuild({
-    minify: true,
+    minify: false,
```

Comment:
Is this correct for all envs / frameworks?

Reply:
fixed